site/_posts/2017-09-25-last-week-of-summer.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Last Week in ManageIQ: Last week of summer ... "
+author: [lgalis](https://github.com/lgalis)
```

Comment:
I don't think links are valid in the front matter. Please remove the URL and the brackets, thanks!

Reply:
@cybette  - Done